src/D2L.CodeStyle.Analyzers/Async/Generator/FileCollector.cs

```diff
@@ -80,6 +80,9 @@ internal sealed class FileCollector {
 		}
 
 		public string CollectSource() {
+			// This allows us to copy+paste annotations but otherwise does not emit diagnostics
+			// otherwise we get "CS8669: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. Auto-generated code requires an explicit '#nullable' directive in source."
+			m_out.AppendLine( "#nullable annotations" );

```

Comment:
list of options: https://learn.microsoft.com/en-us/dotnet/csharp/nullable-references#nullable-contexts